tests/cppunit/types/bitmap_test.cc

```diff
@@ -72,39 +72,56 @@ TEST_P(RedisBitmapTest, BitCount) {
     bitmap_->SetBit(key_, offset, true, &bit);
   }
   uint32_t cnt = 0;
-  bitmap_->BitCount(key_, 0, 4 * 1024, &cnt);
+  bitmap_->BitCount(key_, 0, 4 * 1024, false, &cnt);
   EXPECT_EQ(cnt, 6);
-  bitmap_->BitCount(key_, 0, -1, &cnt);
+  bitmap_->BitCount(key_, 0, -1, false, &cnt);
   EXPECT_EQ(cnt, 6);
   auto s = bitmap_->Del(key_);
 }
 
+TEST_P(RedisBitmapTest, BitCountBitmapString) {
+  std::string str = "hello";
+  string_->Set(key_, str);
```

Comment:
we can avoid hackings for `str = "hello"`, just set specific bits. Setting a string would be hacking

https://github.com/apache/kvrocks/pull/2075 This patch unify their behaviors